src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -325,17 +325,34 @@ public int getSubTrackCount() {
 	}
 
 	public boolean isVideo() {
-		return videoTrackCount > 0;
+		return getMediaType().equals(MediaType.VIDEO);
 	}
 
 	public boolean isAudio() {
-		return videoTrackCount == 0 && audioTracks.size() == 1;
+		return getMediaType().equals(MediaType.AUDIO);
 	}
 
 	public boolean hasAudio() {
 		return audioTracks.size() > 0;
 	}
 
+	public MediaType getMediaType() {
+		if (videoTrackCount > 0) {
```

Comment:
My whole goal when doing it this way was to get away from the "format" concept as many of the container formats can be considered any media type depending on their content.

MP3 isn't a container format though, and I didn't think it could contain anything but the audio except in the tag. If the tag image is parsed as a VIDEO track, I'd say we have a parsing problem.

Can you give me a sample file? If you don't want to make it public, PM me on the forum.